src/commands/Info/stats.js

```diff
@@ -9,6 +9,13 @@ const formatTime = (time) => {
     return `${hours.toFixed(0)}h ${minutes.toFixed(0)}m ${seconds.toFixed(0)}s`;
 };
 
+const ActivityTypes = [
```

Comment:
Please use camelCase instead of PascalCase :)

(change to `activityTypes`)

Reply:
Regarding the official Docs: That is, because it isn't officially released yet. :P